src/olympia/lib/tests/test_git.py

```diff
@@ -860,30 +860,45 @@ def test_get_diff_newline_new_file():
 
     assert len(changes) == 1
     assert changes[0]['new_ending_new_line'] is True
-    assert changes[0]['old_ending_new_line'] is False
+    assert changes[0]['old_ending_new_line'] is True
 
-    hunk_changes = changes[0]['hunks'][0]['changes']
+    assert len(changes[0]['hunks']) == 0
 
-    assert hunk_changes == [
-        {
-            'content': '{"id": "random"}',
-            'new_line_number': -1,
-            'old_line_number': 1,
-            'type': 'delete'
-        },
-        {
-            'content': '\n\\ No newline at end of file',
-            'new_line_number': -1,
-            'old_line_number': 1,
-            'type': 'insert-eofnl'
-        },
-        {
-            'content': '{"id": "random"}',
-            'new_line_number': 1,
-            'old_line_number': -1,
-            'type': 'insert'
-        }
-    ]
+
+@pytest.mark.django_db
+def test_get_diff_eol_changes():
+    addon = addon_factory(file_kw={'filename': 'webextension_no_id.xpi'})
+
+    original_version = addon.current_version
+
+    AddonGitRepository.extract_and_commit_from_version(original_version)
```

Comment:
I don't think you're making use of `original_version` since you're creating a new `parent_version` - both seem to be duplicated to me. Removing one in favor of the other will speed this test up.

Reply:
This is copied from every other test :) It would speed things up yeah, I assume you were doing this to start with a fresh manifest you control entirely.

Reply:
yeah, I'm sure I had a good reason in the first place. Re-reading it, it doesn't make much sense. I can file an issue to remove those duplicates.